src/lavinmq/queue/queue_factory.cr

```diff
@@ -52,5 +54,11 @@ module LavinMQ
     private def self.stream_queue?(frame) : Bool
       frame.arguments["x-queue-type"]? == "stream"
     end
+
+    private def self.check_unsupported_queue_type(frame)
```

Comment:
Could the name of the method be `warn_if_unsupported_queue_type` instead? I think it's easier to understand what it does then
```suggestion
    private def self.warn_if_unsupported_queue_type(frame)
```

Reply:
sure!